src/prism.c

```diff
@@ -8583,8 +8583,10 @@ parse_target(pm_parser_t *parser, pm_node_t *target) {
                     pm_parser_local_add_location(parser, message.start, message.end);
                     pm_node_destroy(parser, target);
 
+                    uint32_t depth = 0;
+                    for (pm_scope_t *scope = parser->current_scope; scope && scope->transparent; depth++, scope = scope->previous);
```

Comment:
I think this might be easier to read as a `while` on 4 lines:
```c
pm_scope_t *scope = parser->current_scope;
while (scope && scope->transparent) {
    depth++;
    scope = scope->previous;
}
```